Gaming The Syllabus/Assets/Scripts/Riddles/SyllabusRiddleManager.cs

```diff
@@ -47,4 +54,17 @@ private static void LoadRiddlesFromXML(string filePathToRiddlesFolder)
             riddles.Add(XmlUtilities.Deserialize<Riddle>(riddlePath.Replace("Assets/", "")));
         }
     }
+
+    public static bool AreAllRiddlesCompleted()
+    {
+        if (riddles != null && solved_riddles != null)
+        {
+            int num_of_riddles = riddles.Count;
+            int num_of_solved_riddles = solved_riddles.Count;
+            return num_of_riddles == num_of_solved_riddles;
```

Comment:
This works fine because HashSets don't allow duplicate elements right?

Reply:
yeah correct, in theory that wouldn't be an issue